packages/playground-poc/public/scripts/node.js

```diff
@@ -0,0 +1 @@
+../../../node/dist/index.iife.js
```

Comment:
is this a symlink? does it work with `express.static`? I tried it before and it didn't but if this works gr8

Reply:
oh cool @mvanderh you can prob do this in the other PR #263

Reply:
neat!